classes/privacy/provider.php

```diff
@@ -0,0 +1,40 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Privacy Subsystem implementation for format_tabtopics.
+ *
+ * @package    report_accessaudit
+ * @copyright  2019 John Yao <johnyao@catalyst-net.au>
```

Comment:
date? 2020 :)

Reply:
Hi @brendanheywood 
This is fixed in commit c9d1841e22374e2e6d31df555e05f79bc195e872